pkg/manager/manager.go

```diff
@@ -318,6 +384,88 @@ func (m *manager) DetachRepresentor(conf *types.PluginConf) error {
 		log.Info().Msgf("Restoring MTU %d on rep %s", conf.OrigRepState.MTU, conf.Representor)
 	}
 
+	// remove vlan config from uplink if configured to do so
+	if conf.SetUplinkVlan {
+		if err = m.deleteUplinkVlans(rep, conf); err != nil {
+			log.Warn().Msgf("Failed to delete trunk VLANs to parent uplink %v", err)
```

Comment:
nit: reword : "Failed to delete trunk VLANs from uplink"

Reply:
done